algorithms/basque/stemmer.tt

```diff
@@ -1,52 +1,54 @@
-[% header('A Basque stemming algorithm') %]
+[% header('Euskera stemming algorithm') %]
```

Comment:
Is there a reason for this change?  We call this stemmer "Basque" everywhere else, and that does seem to be the English name for the language - we don't say "Deutsch stemming algorithm" for the German stemmer.

Reply:
Agree my bad.